### ISSUE TITLE
[hi] Localize Kubernetes for Hindi

### ISSUE THREAD

**Issue by kumarankit999** [cncf/glossary#2458]
Localize Kubernetes in the glossary for Hindi

https://glossary.cncf.io/kubernetes/

**Comment by ARYANis0001** [comment]
Please assign this issue to me I have raised the PR

**Comment by ARYANis0001** [comment]
@kumarankit999 is there any need of improvement

**Comment by kumarankit999** [comment]
Hey @ARYANis0001 have you raised any PR against this issue?

**Comment by ARYANis0001** [comment]
Yes I raised it But @aj11anuj  told me that PR for this issue is already raised by someone

**Comment by kumarankit999** [comment]
can you mention that PR!

**Comment by ARYANis0001** [comment]
Thank you for your willingness to help us in localizing effort. But i want to let you know that this is a duplicate PR, we already have https://github.com/cncf/glossary/pull/1615. So, we cannot accept this one. Make sure there is no open Issue/PR for the content that you want to localize, otherwise your efforts won't be worth it. But don't get disheartened as you can help us in localizing other terms, for which a PR is not raised yet :)

**Comment by bishal7679** [comment]
Closing as this has already been [merged](https://github.com/cncf/glossary/blob/dev-hi/content/hi/kubernetes.md)
